jupyter_db/main.py

```diff
@@ -277,3 +292,55 @@ def add_outputs_from_nb(
                     for j, output in enumerate(nb_cell.outputs)
                 ]
                 orm_cell.execution.outputs = orm_outputs
+
+    def write_mime_to_file(
+        self,
+        pk: int,
+        dir_path: str,
+        *,
+        fname_format="mimebundle_{pk}{ext}",
+        session: Optional[Session] = None,
+    ) -> str:
+        with self.context_session(
+            session=session, final_commit=False
+        ) as session:  # type: Session
+            mime: OrmMimeBundle = session.query(OrmMimeBundle).filter_by(pk=pk).one()
+            source = mime.source
+            if mime.mimetype == "html":
+                source = dedent(source)
```

Comment:
Is this always a safe operation?

Reply:
What do you mean safe? It’s not actually executing the source?

Reply:
Sorry, that's a noob question: is resenting html guaranteed to not change its looks?